File: app/src/main/java/com/walfud/contactsync_android/service/contact/ContactModel.java
package com.walfud.contactsync_android.service.contact;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by walfud on 2017/5/7.
 */

public class ContactModel {
    public boolean isDeleted;
    public String displayName;
    public List<PhoneModel> phoneList = new ArrayList<>();
    public List<EmailModel> emailList = new ArrayList<>();
    public List<String> noteList = new ArrayList<>();
    public List<AddressModel> addressList = new ArrayList<>();
    public List<ImModel> imList = new ArrayList<>();
    public OrganizationModel organization;
}

File: app/src/main/java/com/walfud/contactsync_android/service/contact/ContactModel.kt
package com.walfud.contactsync_android.service.contact

/**
 * Created by walfud on 2017/5/7.
 */

class ContactModel {
    var id: Long = 0
    var isDeleted: Boolean = false
    var modifyTime: Long = 0

    var displayName: String? = null
    var phoneList: List<PhoneModel>? = null
    var emailList: List<EmailModel>? = null
    var noteList: List<String>? = null
    var addressList: List<AddressModel>? = null
    var imList: List<ImModel>? = null
    var organization: OrganizationModel? = null
}
